tests/test_client.py

```diff
@@ -14,6 +15,14 @@
 )
 from crossfire.parser import UnknownFormatError
 
+if importlib.util.find_spec("pandas"):
+    HAS_PANDAS = True
+else:
+    HAS_PANDAS = False
```

Comment:
Is there any reason to adopt a different [pattern than the one already used in the package](https://github.com/search?q=repo%3AFelipeSBarros%2Fcrossfire+HAS_PANDAS&type=code)?